bookmarked.html

```diff
@@ -27,20 +29,22 @@ <h1 class="header__headline">Quiz App</h1>
           </svg>
         </button>
         <span>Will there be a next question?</span>
-        <button class="card__button" class="card__button_show">
+        <button class="card__button" class="card__button--show">
           Show answer
         </button>
         <!--<p class="card__answer">New answer</p>
 <button class="card__button" class="card__button_hide">Hide answer</button> -->
         <ul class="card__list">
-          <li class="card__list__items" class="html">#html</li>
-          <li class="card__list__items" class="flexbox">#flexbox</li>
-          <li class="card__list__items" class="css">#css</li>
+          <li class="card__list--items" class="html">#html</li>
+          <li class="card__list--items" class="flexbox">#flexbox</li>
+          <li class="card__list--items" class="css">#css</li>
         </ul>
```

Comment:
`--` is used for modifiers. As items are elements of a block I think this naming would be better: `card__list` for `ul` and `list__item` for `li`. I would also go with the singular of items. :)